package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "planout",
-  "version": "2.0.2",
+  "version": "3.0.2",
```

Comment:
I'm not sure if this merits a major version bump since it doesn't impact default behavior. I think this should be 2.1.0 but if you have strong opinions on it then at least this should be 3.0.0 not 3.0.2